crates/sui-source-validation-service/tests/tests.rs

```diff
@@ -35,3 +36,24 @@ async fn test_api_route() -> anyhow::Result<()> {
     expected.assert_eq(&json.source);
     Ok(())
 }
+
+#[test]
+fn test_parse_package_config() -> anyhow::Result<()> {
+    let config = r#"
+    [[packages]]
+    repository = "https://github.com/mystenlabs/sui"
+    paths = [
+        "crates/sui-framework/packages/deepbook",
+        "crates/sui-framework/packages/move-stdlib",
+        "crates/sui-framework/packages/sui-framework",
+        "crates/sui-framework/packages/sui-system",
+    ]
+"#;
+
+    let config: Config = toml::from_str(config).unwrap();
+    let expect = expect![
+        r#"Config { packages: [Packages { repository: "https://github.com/mystenlabs/sui", paths: ["crates/sui-framework/packages/deepbook", "crates/sui-framework/packages/move-stdlib", "crates/sui-framework/packages/sui-framework", "crates/sui-framework/packages/sui-system"] }] }"#
```

Comment:
Not sure if there's a prettier way to output this than writing a `Display`, which I figured to do later

Reply:
Fixed this with `:#?`